### PR TITLE
[UI][Server] Update API endpoint call

### LINKED ISSUE #5192: [Server] Not able to remove K8s config
#### Current Behavior
When a user clicks to remove the single-connected cluster, Meshery Server fails to remove the config.

#### Expected Behavior
Meshery Server should be disconnected from the cluster. Operator should be uninstalled.

#### Screenshots/Logs
Screen Shot 2022-03-09 at 10 12 18 AM

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v0.6.0-rc-5b
- **Meshery Client Version:** stable-v0.6.0-rc-5b

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>

**Description**
Updated the endpoint, no more errors on removing(successful request), 

This PR fixes #5192 

**Notes for Reviewers**
This needs some more workaround with the backend/frontend to remove completely.
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — COMMENTED** [review]
Does this only affect the contacts which are in the title bar or does this also affect other chips like that of the one in the settings page?

**Comment by Chadha93** [comment]
> Does this only affect the chips which are in the title bar or does this also affect other chips like that of the one in the settings page?

This will affect only the switcher chip only, though the endpoints and methods will remain the same for both.

File: ui/components/ConnectionWizard/helpers/common.js [hunk 1]
```diff
@@ -28,5 +28,5 @@ export const errorHandlerGenerator = (snackbar, action,msg, cb) => (err) => {
   if (cb !== undefined) cb(err)
   snackbar(`${msg}: ${err}`, { variant : "error",
     action,
-    autoHideDuration : 7000, })
+    autoHideDuration : 2000, })
```
Comment (leecalcote) [inline ui/components/ConnectionWizard/helpers/common.js:31]:
```suggestion
    autoHideDuration : 3000, })
```

File: ui/components/ConnectionWizard/helpers/common.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export const successHandlerGenerator = (snackbar, action, msg, cb) => (res) => {
     if (cb !== undefined) cb(res)
     snackbar(msg, { variant : "success",
       action,
-      autoHideDuration : 7000, })
+      autoHideDuration : 2000, })
```
Comment (leecalcote) [inline ui/components/ConnectionWizard/helpers/common.js:22]:
```suggestion
      autoHideDuration : 3000, })
```

**Review by leecalcote — DISMISSED** [review]

**Pull request merged**